labs/gnmic-prometheus-grafana/playbooks/fabric-deploy-config.yaml

```diff
@@ -0,0 +1,25 @@
+- name: Build cEOS EVPN VXLAN All-active Multi-homing IRB Fabric (eBGP Overlay and eBGP Underlay)
+  hosts: OM_FABRIC
+  tasks:
+    # - name: Generate EOS configuration Snapshots
+    #   tags: [snapshot]
+    #   import_role:
+    #     name: arista.avd.eos_snapshot
+
+    - name: Generate AVD Structured Configurations and Fabric Documentation
+      import_role:
+        name: arista.avd.eos_designs
+
+    - name: Generate Switch Intended Configurations and Documentation
+      import_role:
+        name: arista.avd.eos_cli_config_gen
+
+    - name: Deploy generated configuration to devices
```

Comment:
I'd prefer to have separate playbooks for various steps, [like here](https://github.com/arista-netdevops-community/one-click-se-demos/tree/main/demos/cvaas-cvaas-and-avd-demo--evpn-aa-small/avd_inventory/playbooks). Called by make shortcuts.
Certainly it's fine to have combined build and deploy on top.